extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -312,14 +312,48 @@ export class ProjectsController {
 			let publishDatabaseDialog = this.getPublishDialog(project);
 
 			publishDatabaseDialog.publish = async (proj, prof) => this.publishOrScriptProject(proj, prof, true);
+			publishDatabaseDialog.publishToContainer = async (proj, prof) => this.publishToDockerContainer(proj, prof);
 			publishDatabaseDialog.generateScript = async (proj, prof) => this.publishOrScriptProject(proj, prof, false);
 			publishDatabaseDialog.readPublishProfile = async (profileUri) => readPublishProfile(profileUri);
 
 			publishDatabaseDialog.openDialog();
 
 			return publishDatabaseDialog.waitForClose();
 		} else {
-			return launchPublishDatabaseQuickpick(project, this);
+			void this.publishDatabase(project);
+			return undefined;
```

Comment:
don't need to return anything here since the return type is Promise<void> anyway